DNN Platform/Library/Framework/JavaScriptLibraries/JavaScript.cs

```diff
@@ -80,15 +80,6 @@ public static void RequestRegistration(string jsname)
                 case CommonJs.jQuery:
                     RequestRegistration(CommonJs.jQueryMigrate);
                     break;
-                case CommonJs.DnnPlugins:

```

Comment:
Everything looks good to me but I don't understand why the case statements are removed from here... Can you elaborate on that ?

Reply:
Because these are now being included as JS libraries, there's no more special behavior need, all of this logic is now part of the library itself as dependencies.

Reply:
Sorry, I have not dug much in this area, but do we need to do anything (deprecate) in the CommonJs class or is that still usable in other parts of the code ?

Reply:
It's just a collection of strings for the JS libraries that come with DNN. I don't see a need to deprecate, unless we're planning to stop including a library.